### DISCUSSION TITLE
don't just read this repositorie help me finds bugs

### DISCUSSION THREAD

**Discussion by webcharmer** [webcharmer/LMS#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by webcharmer** [comment]
### ADDING SOME REFERENCES BUTTON TO GET ACCESS ON DATA BASE WITH ADMIN PASS AT THE ADMIN PART

**Reply by saifeddineelhanoune** [reply]
it's seems like you should take a change on where the connection data base created and starting with a helpful analyse to get the admin access first after you should start within because of the first students table i'm not too familiar with python django but this what i have seen

**Reply by webcharmer** [reply]
but i have also made changes on student part firstly and i will lastly fix this missing part thank you

**Reply by saifeddineelhanoune** [reply]
good luck brother

**Reply by webcharmer** [reply]
thanks

**Reply by webcharmer** [reply]
@0x013FA11 you're right thanks for your clear answer

**Comment by webcharmer** [comment]
> it's seems like you should take a change on where the connection data base created and starting with a helpful analyse to get the admin access first after you should start within because of the first students table i'm not too familiar with python django but this what i have seen
